src/components/deposit-hints/DepositHintsModalEdit.js

```diff
@@ -40,7 +40,7 @@ const DepositHintsModalForm = (props) => {
       const parsedValues = parseSubmitValues(values);
       const url = `/user/${user.id}/deposithints`;
       await gpib.secure.put(url, parsedValues);
-      if (userEnterprise) {
+      if (userEnterprise && userEnterprise?.name) {
```

Comment:
@rellfy seems to be the same thing and a redundant check?

Reply:
```suggestion
      if (userEnterprise?.name) {
```